package.json

```diff
@@ -2,14 +2,20 @@
   "name": "h5wasm",
   "version": "0.6.10",
   "description": "A high-level library for reading and writing HDF5 files from Javascript, using a wasm-compiled version of the HDF5 C library",
-  "types": "./src/hdf5_hl.d.ts",
   "type": "module",
-  "main": "./dist/node/hdf5_hl.js",
+  "main": "./dist/iife/hdf5_hl.js",
```

Comment:
It seems unpkg and jsdelivr both [fall back](https://github.com/advename/The-React-Library-guide#210--how-does-a-library-work) to the `browser` field and/or the `main` field, so I'm putting the IIFE build in there. This should allow shortening the URL, for instance:

```js
self.importScripts('https://cdn.jsdelivr.net/npm/h5wasm@0.4.9');
```